cdap-unit-test/src/test/java/co/cask/cdap/test/app/TestAppWithCube.java

```diff
@@ -51,6 +53,10 @@
  *
  */
 public class TestAppWithCube extends TestBase {
+
+  @ClassRule
+  public static final TestConfiguration CONFIG = new TestConfiguration("explore.enabled", false);
```

Comment:
Fixed. For those tests in cdap-example, I am keeping them as string, since it's not supposed to use CDAP internal class in regular app.